packages/serialization/json/src/jsonSerializationWriter.ts

```diff
@@ -75,6 +110,11 @@ export class JsonSerializationWriter implements SerializationWriter {
 		}
 	};
 	public writeCollectionOfObjectValues = <T extends Parsable>(key: string, values: T[], serializerMethod: ModelSerializerFunction<T>): void => {
```

Comment:
```suggestion
	public writeCollectionOfObjectValues = <T extends Parsable>(key: string, values: T[] | null | undefined, serializerMethod: ModelSerializerFunction<T>): void => {
```

(and repeat for other object/enum methods)

Reply:
should I touch other places? `form`, `text`
<img width="205" alt="image" src="https://github.com/microsoft/kiota-typescript/assets/41145554/b73ae6e4-9efd-4899-a4ca-ba03a1c9f8f8">

Reply:
Yes things need to be aligned between the interface season and the implementations.

Reply:
@baywet last on this, should I care about `multipart` serialization?
![image](https://github.com/microsoft/kiota-typescript/assets/41145554/b2553031-de02-4a47-941c-cdc5927405ca)

Reply:
meaning: should you write a null value? or did you ask a different question?

Reply:
yes and add `null` type

Reply:
yes, I think we should both have null in the parameter type, and write null to json when the value is null (but not when it's undefined)

Reply:
@baywet Should I touch the `writeByteArrayValue`?

<img width="458" alt="image" src="https://github.com/microsoft/kiota-typescript/assets/41145554/95deeff6-4a98-4da1-93d9-449fc0078e26">

Reply:
@baywet I think I did it, please review
![image](https://github.com/microsoft/kiota-typescript/assets/41145554/cf0fa56f-3df5-4c48-bd09-2c7156c9bc11)